src/components/workouts/ExerciseLibrary/ExerciseView.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react'
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import { Exercise } from "./ExerciseLibrary.types";
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { useNavigate } from 'react-router-dom';
+
+const ExerciseView = (props: { exercise: Exercise }) => {
+
+    const navigate = useNavigate();
+
+    return (
+        <Box
+            sx={{
+                position: "fixed",
+                top: 0,
+                left: 0,
+                width: "100vw",
+                height: "100vh",
+                bgcolor: 'primary',
+                zIndex: 1000,
+                padding: 2,
+            }}
+        >
+
+
+            <Box
+                sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                }}
+            >
+                <Button variant="text" color="primary" startIcon={<ArrowBackIcon />} onClick={() => navigate(`/Exercise-Library`)}>
```

Comment:
considering the feedback we got from the co-heuristics, we may want to stick with a consistent format for the back arrow that we use in other pages